data/json/items/armor/ammo_pouch.json

```diff
@@ -598,7 +599,11 @@
     "symbol": "[",
     "color": "dark_gray",
     "material_thickness": 2,
-    "use_action": { "type": "holster", "holster_prompt": "Stash item", "holster_msg": "You stash your %s." },
+    "use_action": [
+      { "type": "holster", "holster_prompt": "Stash item", "holster_msg": "You stash your %s." },
+      { "type": "attach_molle", "size": 3 },
```

Comment:
The size is likely incorrect. My understanding is that 6 cells is 1 slot. At most I can imagine 9 usable cells on that small patch, with many of them well outside of the comfortable pouch-attaching range.

Even then, it's unlikely you would be able to fit in more than one slot's worth of pouches, and since 1 slot is the minimum size, that's probably what you'd round that down to. 2 slots is for cases where you want to be generous, with this vest.

Reply:
We are talking about the MOLLE attachment slots, correct?  Are you taking into account the cells on the back? Because there are 30 cells on the back alone.

I wasn't really sure how to equate cells to slots, so for figuring slots on the back I figured half of what the light load bearing vest had, which is 4, but if it's one slot for 6 cells then maybe It should be increased?

Reply:
![Uploading image.png…]()

Reply:
The way I've been modelling MOLLE gear in [Armory](https://github.com/FrontierMods/Armory/) is under the impression that you are not using those nearly as easily as your front and side webbing. Even if you can fit a mag pouch or a sheath or a gas mask pouch there, once the vest is on you, your arm is nowhere near bendy enough to grab something out of those pouches with the grace it would from a chest pouch.

Back webbing is mostly used for pouches that *others* grab from – like, say, your squad members. Each member, then, would serve as a carry for extra supplies that other specialists (medic, grenadier, comms...) would use. The only useful-to-self back-webbing pouches I've seen are [breaching tool pouches](https://www.bushidotactical.com/product/bt-molle-soft-shell-collapsible-breaching-tool-pouch/), [small-of-the-back hanging magazine pouches](https://www.cryeprecision.com/Side-Pull-Mag-Pouch) (which I'm not positive is a *MOLLE* pouch), and [a hydration-pack panel](https://www.qoreperformance.com/en-eu/products/iceplate-molle-sleeve). [The rest](https://hrttacticalgear.com/back-panels/) are, all in all, for your squad's use.

Even all of that aside, that's at least 5 slots. 6 if you want to be generous with those extra-wide side cells.

Reply:
I agree with you that the real estate back there is not terribly accessible for oneself, so might just leave it at two slots if no one has an issue with it, for the left and right side.